app/src/main/java/com/github/se/polyfit/model/post/Location.kt

```diff
@@ -14,5 +14,13 @@ data class Location(
     fun default(): Location {
       return Location(0.0, 0.0, 0.0, "EPFL")
     }
+
+    fun deserialize(data: Map<String, Any>): Location {
+      return Location(
+          data["longitude"] as Double,
+          data["latitude"] as Double,
+          data["altitude"] as? Double ?: 0.0,
```

Comment:
Where do we need the altitude in the app?

Reply:
It is artifact of a previous version used to avoid breaking anything when fetching data from the firebase.